src/IMachineLearningModel.ts

```diff
@@ -0,0 +1,9 @@
+import * as Blockly from 'blockly/core';
+
+interface MachineLearningModel {
+  getSuggestedBlocks(description: string): Blockly.Block[];
```

Comment:
In order to return an Array of blocks, we'll need a [`Workspace`](https://github.com/google/blockly/blob/develop/core/workspace.ts), so let's add a parameter variable of this type.